codec-http/src/main/java/io/netty/handler/codec/http/DefaultLastHttpContent.java

```diff
@@ -107,32 +108,24 @@ private void appendHeaders(StringBuilder buf) {
     }
 
     private static final class TrailingHttpHeaders extends DefaultHttpHeaders {
-        private static final class TrailingHttpHeadersNameConverter extends HttpHeadersNameConverter {
-            TrailingHttpHeadersNameConverter(boolean validate) {
-                super(validate);
-            }
+        private static final class TrailingHttpHeadersNameValidator implements
+                                                                    DefaultHeaders.NameValidator<CharSequence> {
+
+            public static final TrailingHttpHeadersNameValidator INSTANCE = new TrailingHttpHeadersNameValidator();
```

Comment:
https://github.com/netty/netty/pull/3681#discussion_r35021121 was collapsed...

Why is this `public`?  I don't think the intention is to access it externally to this class.  Consider making `private`.